JDP191202-20

naros1 opened the pull request:
Pierwszy Test relacji encji User z Cart oraz Order dodany. Pozytywnym wynikiem 

Poprawki dodane w encji UserEntity dla List<OrderEntity> orders na List<Order> orders = new ArrayList<>();
Błędy w encjach CartEntity brak <... producs= new ArrayList<>(); > | Group (w atrybuach kostruktora niepotrzebna lista products) | Product (niepotrzebny atrybut group w kostruktorze i brak <... carts= new ArrayList<>(); >

naros1 committed:
Pierwszy Test relacji encji User z Cart oraz Order dodany. Pozytywnym wynik
Poprawki dodane w encji UserEntity dla List<OrderEntity> orders na List<Order> orders = new ArrayList<>();
Błędy w encjach CartEntity brak <... producs= new ArrayList<>(); > | Group (w atrybuach kostruktora niepotrzebna lista products) | Product (niepotrzebny atrybut group w kostruktorze i brak <... carts= new ArrayList<>(); >

Nikoguj commented on src/test/resources/application.properties:
```diff
@@ -1,4 +1,15 @@
+#spring.jpa.show-sql=true
+#spring.jpa.hibernate.ddl-auto=update
+#spring.jpa.hibernate.naming.strategy=org.hibernate.cfg.ImprovedNamingStrategy
+#spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
```
Raczej nie jest to potrzebne

naros1 replied:
przydaje się jak chchemy przeprowadzić testy dodatkowo w bazie mySQL ja je robiłem i będe jeszcze robił w dalszych testach dlatego zostawiam

naros1 replied:
już testy poprawion przechodzą Circleci

naros1 replied:
https://circleci.com/gh/klikuskodilla/JDP-1912-02/73?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Nikoguj reviewed (changes requested):
Pull request nie przechodzi Circleci https://circleci.com/gh/klikuskodilla/JDP-1912-02/72?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

naros1 committed:
Pierwszy Test relacji encji User z Cart oraz Order dodany. Pozytywnym wynik
Poprawki dodane w encji UserEntity dla List<OrderEntity> orders na List<Order> orders = new ArrayList<>();
Błędy w encjach CartEntity brak <... producs= new ArrayList<>(); > | Group (w atrybuach kostruktora niepotrzebna lista products) | Product (niepotrzebny atrybut group w kostruktorze i brak <... carts= new ArrayList<>(); >

klikuskodilla commented on src/test/resources/application.properties:
```diff
@@ -1,4 +1,15 @@
\ No newline at end of file
+spring.datasource.password=sa
+
+#spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False
```
dlaczego to się pojawiło ?

naros1 replied:
robiłem dodatkowe testy w MySql aby podejrzeć w workbenchu czy wszystkie id się zgadzają. Usunąć ?

naros1 replied:
Usunięte

klikuskodilla commented on src/test/java/com/kodilla/ecommercee/domain/user/dao/UserDaoTestSuite.java:
```diff
@@ -45,4 +59,57 @@ public void testUserDaoSave(){
+        Assert.assertEquals(readOrder2Adress, "Adress 2");
+        Assert.assertEquals(BigDecimal.valueOf(30.99), readCart);
+        Assert.assertEquals(readCartUser, "Test Login 2");
+        System.out.println(
```
Usuń wszystkie wywołania System.out.print i przenieś do asercji

naros1 replied:
poprawione dodane Assercje

klikuskodilla reviewed (changes requested):
Usuń pusty test contextLoads, posiadanie go w każdej klasie testowej jest zbędne ponieważ sprawdzanie spójności aplikacji wykonuję się w innych testach

klikuskodilla commented on src/test/java/com/kodilla/ecommercee/domain/user/dao/UserDaoTestSuite.java:
```diff
@@ -8,14 +14,22 @@
 public class UserDaoTestSuite {
     @Autowired
     private UserEntityDao userDao;
+    @Autowired
```
ProductDao i CartEntityDao są nieużywane

naros1 replied:
poprawione

naros1 committed:
changes requested by klikuskodilla applied

naros1 committed:
changes requested by klikuskodilla applied

naros1 committed:
changes requested by klikuskodilla applied

klikuskodilla commented:
W treści commita powinno zostać wyjaśnione jakie konkretnie zmiany zaszły
![image](https://user-images.githubusercontent.com/33075397/71825221-abe1fd80-309b-11ea-9893-268ecf660b5a.png)

klikuskodilla commented:
Przed zrobieniem pusha możesz uruchomić testy lokalnie żeby uniknąć takiej sytuacji
np:
` gradlew.bat build`

![image](https://user-images.githubusercontent.com/33075397/71825337-f2cff300-309b-11ea-8377-395119b6e000.png)
